assets/js/googlesitekit/datastore/user/feature-tours.test.js

```diff
@@ -26,7 +26,8 @@ import {
 } from '../../../../../tests/js/utils';
 import * as CacheModule from '../../../googlesitekit/api/cache';
 import { STORE_NAME } from './constants';
-import initialState, {
+import {
+	initialState,
```

Comment:
I'm surprised this worked at all! It was importing the default export but `eslint-plugin-import` warned me
`warning  Using exported name 'initialState' as identifier for default export  import/no-named-as-default`

Reply:
Probably a weird auto import that wasn't caught. Thanks!